content/intro-to-storybook/vue/es/using-addons.md

```diff
@@ -5,166 +5,86 @@ description: 'Aprende a integrar y usar complementos usando un ejemplo popular'
 commit: '45b6600'
 ---
 
-Storybook cuenta con un sistema robusto de [complementos](https://storybook.js.org/docs/vue/configure/storybook-addons) con el que puede mejorar la experiencia del desarrollador para
-todos en tu equipo. Si ha seguido este tutorial linealmente, hasta ahora hemos hecho referencia a varios complementos, y ya habrá implementado uno en el [Testing](/intro-to-storybook/vue/es/test/).
+Storybook cuenta con un sistema robusto de [complementos](https://storybook.js.org/docs/vue/configure/storybook-addons) con el que puede mejorar la experiencia del desarrollador para todos en tu equipo.Véalos todos [aquí](https://storybook.js.org/addons).
```

Comment:
@IvanSotelo could you add a space here turning this into:

"
... con el que puede mejorar la experiencia del desarrollador para todos en tu equipo. Véalos ...."